docs/general/attributes.md

```diff
@@ -45,6 +49,13 @@ This also covers UDP network interactions where one side initiates the interacti
 In an ideal situation, not accounting for proxies, multiple IP addresses or host names,
 the `server.*` attributes are the same on the client and server.
 
+### Address and port attributes
+
+For the purpose of these semantic conventions, for all IP-based protocols the "address" should be just IP-level address,
```

Comment:
```suggestion
For all IP-based protocols, the "address" should be just the IP-level address.
```
I'm not sure we need the "For the purpose of these semantic conventions" part. Also since this is general/attributes.md, it might be confusing as to what "these semantic conventions" refer to.